testcases/kafka-broker-ordered/100-broker.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  annotations:
+    # case-sensitive
+    eventing.knative.dev/broker.class: Kafka
+  name: my-kafka-broker
```

Comment:
could we name that differently ? 

all brokers have "unique" names

Reply:
Thanks, done!